sentences/en/lock_HassGetState.yaml

```diff
@@ -3,34 +3,43 @@ intents:
   HassGetState:
     data:
       - sentences:
-          - "is <name> {lock_states:state} [in <area>]"
+          - "(is|are) <name> {lock_states:state} [in <area>]"
         response: one_yesno
         requires_context:
           domain: lock
         slots:
           domain: lock
 
       - sentences:
-          - "(is|are) [there] any door[s] {lock_states:state} [in <area>]"
-          - "(is|are) [there] any {lock_states:state} door[s] [in <area>]"
+          - "(is|are) [there] any (door|lock|window)[s] {lock_states:state} [in <area>]"
+          - "(is|are) [there] any {lock_states:state} (door|lock|window)[s] [in <area>]"
+          - "do I have any {lock_states:state} (door|lock|window)[s] [in <area>]"
         response: any
         slots:
           domain: lock
 
       - sentences:
-          - "are all [the] door[s] {lock_states:state} [in <area>]"
+          - "(is|are) all [the] (door|lock|window)[s] [securely] {lock_states:state} [in <area>]"
         response: all
         slots:
           domain: lock
 
       - sentences:
-          - "(which|what) door[s] (is|are) {lock_states:state} [in <area>]"
+          - "(which|what) (door|lock|window)[s] (is|are) [currently] {lock_states:state} [in <area>]"
```

Comment:
Why would we add "currently" only to this sentence and "securely" only to the previous one? Would it not be better to alter the `lock_state` list in order to optionally match these 2 adverbs as well?

Reply:
Maybe, but `currently` is an adjective that could probably apply to _any_ query. Do you know how `skip_words` are used? They look like they are currently words that may show at the beginning of a query, however there are probably a lot of adverbs that could be added to the list.

* currently
* securely
* presently
* already

Reply:
`skip_words` are simply stripped from the input sentence before any matching is done. I'd not add adverbs to that list, as common as they may be

Reply:
In that case, possibly an `adverbs` expansion would be useful? I can see your point to not stripping though as that's probably more blunt. It's possible someone may have an adverb in a script name or something.

Reply:
Made some changes to address this. One thing I don't quite like is that to make sentences flexible, it requires two sentences for each intent. One where the adverb is at the front and one where it's at the end. I'm not sure there's a good way around that to make something even more generic.

Reply:
[Permutations](https://developers.home-assistant.io/docs/voice/intent-recognition/template-sentence-syntax#sentence-templates-syntax), perhaps?

Reply:
Giving that a try!